components/StyledUpdate.js

```diff
@@ -241,13 +245,19 @@ class StyledUpdate extends Component {
   }
 
   renderFullContent() {
-    const { update, collective, isReloadingData } = this.props;
+    const { update, collective, isReloadingData, reactions } = this.props;
 
     return (
       <Container css={{ wordBreak: 'break-word' }} pl={[0, 60]} maxWidth={676}>
         <StyledHr mt={3} mb={4} borderColor="black.100" />
         {update.html ? (
-          <HTMLContent content={update.html} />
+          <React.Fragment>
+            <HTMLContent content={update.html} />
+            <Flex mt={3} flexWrap="wrap" data-cy="update-reactions">
+              {reactions && update.publishedAt && <CommentReactions reactions={reactions} />}
+              {update.publishedAt && <CommentReactionPicker update={update} />}
+            </Flex>
```

Comment:
Seems none of this is going to be displayed if the update is not published, maybe we can simplify and skip the empty block:

```suggestion
            {update.publishedAt && (
              <Flex mt={3} flexWrap="wrap" data-cy="update-reactions">
                {reactions && <CommentReactions reactions={reactions} />}
                <CommentReactionPicker update={update} />
              </Flex>
            )}
```